runtime/doc/vim_diff.txt

```diff
@@ -30,8 +30,11 @@ Defaults					            *nvim-defaults*
   ":filetype off" to |init.vim|.
 - Syntax highlighting is enabled by default. This can be disabled by adding
   ":syntax off" to |init.vim|.
-- Default color scheme has been updated. This can be reversed by adding
-  ":colorscheme vim" to |init.vim|.
+- Default color scheme has been updated. In particular it might result into
+  color schemes looking differently in Nvim. It is due to them relying on
+  implicit assumptions of how highlight groups are defined by default.
+  This can be reversed by adding ":colorscheme vim" to |init.vim| or color
+  scheme file.
```

Comment:
```suggestion
- Default color scheme has been updated. This can result in color schemes 
  looking differently due to them relying on how highlight groups are defined 
  by default. Add ":colorscheme vim" to |init.vim| or your color scheme file to
  restore the old default links and colors.
```